crates/biome_js_parser/src/lexer/mod.rs

```diff
@@ -827,22 +827,32 @@ impl<'src> JsLexer<'src> {
     /// This method will stop writing into the buffer if the buffer is too small to
     /// fit the whole identifier.
     #[inline]
-    fn consume_and_get_ident(&mut self, buf: &mut [u8]) -> (usize, bool) {
+    fn consume_and_get_ident(&mut self, buf: &mut [u8]) -> (usize, bool, bool) {
```

Comment:
To update the documentation of the function, to describe the new tuple

Reply:
I've this reverted changes because there is a logic which allows to write escaped unicode for the identifiers.